content/learn/vp_agof.adoc

```diff
@@ -0,0 +1,381 @@
+
+== Role of the Ansible Controller
+
+In the Ansible GitOps framework, an ansible controller refers to any machine that initiates and runs a playbook. This includes systems running ansible core or open source Ansible, where the machine executing the playbooks also acts as the controller. AAP provides the automation controller which is Red Hat's productized ansible controller. 
+
+THe link:https://docs.ansible.com/platform.html[Automation controller] is the command and control center for Red Hat Ansible Automation Platform, replacing Ansible Tower. It includes a webUI, API, role-based access control (RBAC), a workflow visualizer, and continuous integration and continuous delivery (CI/CD) integrations to help you organize and manage automation across your enterprise.
+
+The Automation Controller serves as the centralized platform for managing and executing automation across infrastructure. Use the Automation Controller to create job templates that standardize the deployment and execution of Ansible playbooks, making automation more consistent and reusable. It integrates essential components such as execution environments for consistent automation execution, projects (repositories for automation content), inventories (target endpoints), and credentials (for secure access to resources).
+
+The webUI provides an intuitive interface to build, monitor, and manage automation workflows, while the API offers seamless integration with other tools, such as CI/CD pipelines or orchestration platforms. Overall, the Automation Controller streamlines the automation lifecycle, ensuring a scalable, secure, and maintainable automation environment.
+
+== Ansible framework methods
+
+The three main methods for setting up an Ansible framework are as follows:
+
+=== Method 1: AWS-based install
+
+This method is ideal for organizations that prefer deploying AAP on AWS infrastructure. This default install process in AAP 2.4 uses AWS by default and offers a fully automated setup. It requires AWS credentials, builds an AWS image with Red Hat's ImageBuilder, and sets up AAP within an AWS VPC and subnet. The installation program creates all the necessary resources, including AAP Controllers and, optionally, additional components such as the Automation Hub.
+
+This is the easiest method if you already use AWS, as it automates the provisioning of resources, including VMs and network configurations. This requires AWS infrastructure and credentials.
+
+=== Method 2: Pre-configured VMs Install
```

Comment:
I am wondering if I shouldn't do a bit more work to improve this section. The last vestiges of legacy AGOF still run here (i.e. the pre-containerized install bits) so the install works a bit differently. Alternatively we could just discuss the AWS install and the API install for now.

Reply:
I might leave out the AWS was the only one I tried so really don't know much about this one.